Source/WebKit/UIProcess/API/glib/WebKitSettings.cpp

```diff
@@ -1713,6 +1720,24 @@ static void webkit_settings_class_init(WebKitSettingsClass* klass)
         nullptr, // A null string forces the default value.
         readWriteConstructParamFlags);
 
+    /**
+     * WebKitSettings:inspector-maximum-resources-content-size:
+     *
+     * Allow customization of the maximum size of the stored resources content by the inspector.
+     *
+     * In case of memory limited environments, this setting can be used to minimize memory
+     * consumption by the inspector. The upper limit of this value is 200MB and lower limit is 50MB.
+     * By default the maximum resources content size is 200MB.
```

Comment:
"In memory-limited environments, this property can be used to reduce the web inspector's memory usage. The value specified is in megabytes, and its valid range goes from 50 to 200 megabytes."

I think you don't need to specify the default value, as that will be done by the documentation generator.